app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+const {
+  getAllTopics,
+} = require("./controllers/news.controllers");
+
+app.use(express.json());
```

Comment:
^^ You don't need this in yet - it's only useful when you start sending a body (i.e. a post or a patch). Please remove it from your next branch :)